test/parallel/xyk-pallet.asset.register.test.ts

```diff
@@ -196,7 +196,7 @@ test("register asset and then try to register new one with the same assetId, exp
 });
 
 // needs to be moved to sequential, some other parallel test might use the same asset id resulting in "ConflictingAssetId" error instead
-test.skip("register asset and then try to register new one with the same location, expect to conflict", async () => {
+test("register asset and then try to register new one with the same location, expect to conflict", async () => {
```

Comment:
Plz, move this test to a sequential test run. From the comment above ( L198) it seems that it could cause some Asset conflicts